qml/NewEvent.qml

```diff
@@ -543,15 +543,15 @@ Page {
                 }
             }
 
-            ListItems.ThinDivider {}
+            ListItems.ThinDivider {color:theme.palette.normal.backgroundTertiaryText}
```

Comment:
This change too seems unrelated. What is the reason for this change?

Reply:
I **think** this is here as in Suru Dark the default color for the ThinDivider is hidden (same color as the background) I`ll have a better answer tomorrow...

Reply:
Ok no reason for this change  we get the  same result...  I`l revert it

Reply:
Thanks! But if you think that there's some mistake in the Suru dark theme, feel very welcome to submit a branch against the ubuntu-ui-toolkit :-)